tests/Aspire.Hosting.Azure.Tests/AzureServiceBusExtensionsTests.cs

```diff
@@ -801,4 +807,65 @@ public void AzureServiceBusAppliesAzureFunctionsConfiguration()
             k => Assert.Equal("Aspire__Azure__Messaging__ServiceBus__sub__FullyQualifiedNamespace", k),
             k => Assert.Equal("sub__fullyQualifiedNamespace", k));
     }
+
+    [Fact(Skip = "Azure ServiceBus emulator is not reliable in CI - https://github.com/dotnet/aspire/issues/7066")]
+    [RequiresDocker]
+    public async Task AzureServiceBusEmulator_WithCustomConfig_HasHealthChecks()
```

Comment:
This test doesn't do anything with "health checks".

```suggestion
    public async Task AzureServiceBusEmulator_WithCustomConfig()
```